benchmark/src/util.h

```diff
@@ -146,32 +150,32 @@ static inline void put_unaligned_be64(u64 v, void *p)
 
 static inline u16 rol16(u16 word, unsigned int shift)
 {
-	return (word << shift) | (word >> (16 - shift));
+	return (word << shift) | (word >> (-shift & 15));
```

Comment:
What's this change for? Surprised you mask only the second shift in each case.

Reply:
As per the commit message for this change, this is specifically to avoid undefined behavior when rotating by 0 bits.  I have expanded the explanation in the commit message.